CHANGELOG.md

```diff
@@ -43,6 +43,8 @@
 - fix selected account not getting scrolled into view in accounts bar on app start #4542
 - message list being empty sometimes when a chat gets opened #4556
 - improve performance a little #4561, #4552, #4584
+- fix pasting of (non image) files into composer #4533
+- fix translation PluralRules fallback to 'en' not to 'en-US' (en-US isnt valid) #4585
 
```

Comment:
Should we add changelog entries also for the other features?

Reply:
tauri version is not released yet, so I thought we could skip them, but we could also do that